plugins/lsp/fidget-nvim.nix

```
@@ -0,0 +1,240 @@
+{
+  config,
+  pkgs,
+  lib,
+  ...
+}:
+with lib; let
+  cfg = config.plugins.fidget-nvim;
+  helpers = import ../helpers.nix {inherit lib;};
+  plugin-defs = import ../plugin-defs.nix {inherit pkgs;};
+in {
+  options = {
+    plugins.fidget-nvim =
```

```suggestion
    plugins.fidget =
```
I would have this plugin renamed `fidget`.